www/includes/easyparliament/templates/html/mp/_division_description.php

```diff
@@ -1,8 +1,6 @@
 <li id="<?= $division['division_id'] ?>" class="<?= $division['strong'] || $show_all ? 'policy-vote--major' : 'policy-vote--minor' ?>">
     <span class="policy-vote__date">On <?= strftime('%e %b %Y', strtotime($division['date'])) ?>:</span>
     <span class="policy-vote__text"><?= $full_name ?><?= $division['text'] ?></span>
-    <?php if ( $division['url'] ) { ?>
-        <a class="vote-description__source" href="<?= $division['url'] ?>">Show full debate</a>
-    <?php } ?>
+    <a class="vote-description__source" href="/divisions/division.php?vote=<?= $division['division_id'] ?>&p=<?= $person_id ?>">Show vote</a>
```

Comment:
Can we have slightly nicer URLs than these? `/divisions/ID` if the ID is unique, I guess?